packages/extension-ui/src/Popup/Accounts/index.tsx

```diff
@@ -19,58 +21,72 @@ interface Props extends ThemeProps {
   className?: string;
 }
 
-function Accounts ({ className }: Props): React.ReactElement {
+function Accounts({ className }: Props): React.ReactElement {
   const { t } = useTranslation();
   const [filter, setFilter] = useState('');
   const [filteredAccount, setFilteredAccount] = useState<AccountWithChildren[]>([]);
   const { hierarchy } = useContext(AccountContext);
   const networkMap = useMemo(() => getNetworkMap(), []);
 
   useEffect(() => {
-    setFilteredAccount(
-      filter
-        ? hierarchy.filter((account) =>
-          account.name?.toLowerCase().includes(filter) ||
-          (account.genesisHash && networkMap.get(account.genesisHash)?.toLowerCase().includes(filter))
-        )
-        : hierarchy
-    );
+    setFilteredAccount(filter ? hierarchy.filter((account) => account.name?.toLowerCase().includes(filter) || (account.genesisHash && networkMap.get(account.genesisHash)?.toLowerCase().includes(filter))) : hierarchy);
   }, [filter, hierarchy, networkMap]);
 
   const _onFilter = useCallback((filter: string) => {
     setFilter(filter.toLowerCase());
   }, []);
 
+  const _handleMenuCardClick = useCallback(() => {
+    console.log('clicked');
```

Comment:
Please add some `todo: temporary` comment when leaving debug console logs in PRs